docs/tutorials/geomesa-quickstart-lambda.rst

```diff
@@ -125,17 +231,35 @@ start finishes, all the feature should be returned from the persistent store (Ac
 Looking at the Code
 -------------------
 
+The source code is meant to be accessible for this tutorial. The logic is contained in
+the generic ``org.geomesa.example.quickstart.GeoMesaQuickStart`` in the ``geomesa-quickstart-common`` module,
+and the Kafka-specific ``org.geomesa.example.lambda.LambdaQuickStart`` in the ``geomesa-quickstart-lambda`` module.
```

Comment:
Do we want to say `lambda-specific` or `kafka/accumulo specific` or something here?

Reply:
changed to "Kafka/Accumulo-specific"